crates/prover/src/components/mod.rs

```diff
@@ -18,27 +18,32 @@ use stwo_prover::core::pcs::TreeVec;
 use stwo_prover::core::poly::circle::CircleEvaluation;
 use stwo_prover::core::poly::BitReversedOrder;
 
+use crate::preprocessed::range_check;
 use crate::relations;
 
 pub struct Claim<const N: usize> {
     pub single_constraint: single_constraint::Claim<N>,
     pub multiple_constraints: multiple_constraints::Claim<N>,
     pub single_constraint_with_relation: single_constraint_with_relation::Claim,
     pub memory: memory::Claim,
+    pub range_check: range_check::Claim,
```

Comment:
```suggestion
    pub range_check_20: range_check_20::Claim,
```

and subsequent usages of the name `range_check`